tests/test_zip_download.py

```diff
@@ -44,4 +56,4 @@ def test_Download_PNG(test_client):
 def test_zip_file_integrity(test_client):
     test_zip = zipfile.ZipFile(current_app.config['OUTPUT_ZIP'])
     #zip file numbers check
-    assert len(test_zip.namelist()) > 1    
\ No newline at end of file
+    assert len(test_zip.namelist()) == 18    
```

Comment:
Mind clarifying why 18 here?

Reply:
this test uses `tests/testFiles/ActorPreludeSample.musicxml` as the processed file, which should produce 18 files in the zip